import "common.proto";

// travel 831
message TravelRequest{
    required bytes stage_id = 1;
}
message TravelResponse{
    required CommonResponse res = 1;
    required int32 event_id = 2;
    optional int32 time = 3;
}

// travel_init 830
message TravelInitResponse{
    required CommonResponse res = 1;
}
